Bio/SeqFeature.py

```diff
@@ -1537,15 +1562,17 @@ def position(self):
 
     @property
     def extension(self):
-        """Legacy attribute to get extension (from left to right) as an integer (OBSOLETE)."""
+        """Legacy attribute to get the dimension (from left to right) as an integer (OBSOLETE)."""
```

Comment:
I can't due to flake8. It doesn't like when the signature is used inside the summary docstring.